incubator/hnc/test/e2e/demo_test.go

```diff
@@ -87,4 +92,37 @@ var _ = Describe("Demo", func() {
 		RunShouldContain(nsService1, 2, "kubectl hns describe", nsTeamA)
 		RunShouldNotContain("my-creds", 2, "kubectl -n", nsService1, "get secrets")
 	})
+
+	It("Should create and delete subnamespaces", func(){
```

Comment:
More commenting would be nice - it doesn't have to be _as_ detailed as what's in the demo but things like the following might be nice every few lines:

```
// set up an initial structure
// show that you can't re-use a subns name
// delete a subns directly
```

etc

Reply:
done